addons/l10n_fr_account/models/account_move.py

```diff
@@ -16,6 +16,20 @@ def _get_view(self, view_id=None, view_type='form', **options):
                 shipping_fields[0].attrib.pop("groups", None)
         return arch, view
 
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'FR':
```

Comment:
Let's use l10n_fr_is_company_french instead :eyes: Check why and tell me :smile: